src/domains/gps-app/api/payments-service/v1/rest/internal/_base_policy.xml

```diff
@@ -0,0 +1,27 @@
+<policies>
+    <inbound>
+        <set-variable name="requestPath" value="@(context.Request.Url.Path)" />
+        <set-variable name="sub" value="@(context.Subscription.Name)" />
+        <set-variable name="subid" value="@(context.Subscription.Id)" />
+        <set-variable name="subkey" value="@(context.Subscription.Key)" />
+        <set-variable name="organizationFiscalCode" value="@{
+            return context.Variables.ContainsKey("User.Id") ? context.User.Id : "Unknown" ;
+        }" />
+        <base />
+        <!--
+            subset/restriced api of src/core/api/gpd_api/payments/v1/rest/_openapi.json.tpl (INTERNAL) without /payments/{organizationfiscalcode}
+        -->
+        <rewrite-uri template="@("/payments/"+(string)context.Variables["organizationFiscalCode"]+"/"+(string)context.Variables["requestPath"])" copy-unmatched-params="true" />
```

Comment:
why rewrite uri ?

Reply:
didn't we say that rewrite should become a check, like [that](https://github.com/pagopa/pagopa-infra/blob/a43bcf0bffe58375bcbc1474378bfd6512ccc3a5/src/core/api/apiconfig_api/auth/v1/_base_policy.xml.tpl#L37) ?